src/index.html

```diff
@@ -10,8 +10,219 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="field">
+
+          <label class="label">
+          Surname:
+            <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            Name:
+            <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+          How old are You?
+            <input
+            type="number"
+            name="name"
+            value="12"
+            min="1"
+            max="100"
+            required
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+
+          <label class="label">
+            Full date of birth:
+            <input
+            type="date"
+            name="date of birth"
+            required
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            I accsept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              required
+            >
+          </label>
+
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="field">
+
+          <label class="label">
+          E-mail:
+            <input type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            Password:
+            <input type="password"
+            required
+            minlength="5"
+            maxlength="10"
+            >
+          </label>
+
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting facts about you!</legend>
+
+        <div class="field">
+          Do you love cats?
+
+          <label class="label">
+            <input type="radio"
+            name="love cats"
+            value="yes"
+            >
+            Yes
+          </label>
+
+          <label class="label">
+            <input type="radio"
+            name="love cats"
```

Comment:
```suggestion
            name="loveCats"
```
[checklist p.4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) use camelCase for values of name attribute